File: divisa/app/src/main/java/com/example/divisa/ApiService.java
package com.example.divisa;  // Ajusta este paquete al que estés utilizando

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface ApiService {
    @GET("latest/MXN")
    Call<ExchangeResponse> getExchangeRate();
}

File: divisaprovider/app/src/main/java/com/example/divisaprovider/ApiService.kt
package com.example.divisaprovider

import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
    @GET("8ea5b29a23f6bac194ef94dd/latest/MXN")  // 🔹 SOLO la parte final de la URL
    fun getExchangeRates(): Call<ExchangeResponse>
}
